tensorflow/training/docker/2.3/py3/Dockerfile.cpu

```diff
@@ -125,7 +125,7 @@ RUN ${PIP} install --no-cache-dir -U \
     scipy==1.5.2 \
     scikit-learn==0.23 \
     pandas==1.1 \
-    Pillow==7.2.0 \
+    Pillow==8.2.0 \
```

Comment:
The PR description says you're unpinning it, but it looks like the code change just upgrades the version being installed.

Is the decision here to upgrade the version, or to unpin it?

Reply:
changed description